module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -1083,13 +1083,13 @@ public function getConfig($function, $params = null)
      * @return bool True if the method can be called with the given parameters,
      * false otherwise.
      */
-    public function supportsMethod($method, $params)
+    public function supportsMethod(string $method, array $params = [])
```

Comment:
`$params` is not optional, so it shouldn't have a default value.

Reply:
I removed the default value.